batch.py

```diff
@@ -102,13 +103,35 @@ def parse_args():
     return parser.parse_args()
 
 
-def process_feed(args, callback=None):
+def open_connection():
+    """ open connection to database, initialize tables if they don't exist
+        :out:
+            :cursor: interactive sql object containing tables
+    """
+    conn = psycopg2.connect(host="localhost", dbname="gsaid_db", user="postgres",
+                            password="12345", port="5432")
```

Comment:
Passowrd, database name, etc. should be an environment variable. For example, https://github.com/PoonLab/covizu/blob/4ab1ba45b919a45d974c0dc58bf64f09c7dfadb1/batch.py#L20-L23